avm/client/src/avmHelpers.ts

```diff
@@ -162,6 +162,9 @@ export async function callAvm(
         return {
             retCode: -1,
             errorMessage: 'marine-js call failed, ' + e,
-        } as any;
+            data: new Uint8Array(),
+            nextPeerPks: [],
+            callRequests: [],
+        };
```

Comment:
Now any errors from AVM call will be silently ignored? Or are they reported somewhere else?

Reply:
No, fjs will just not fail trying to parse undefined data

Reply:
I've removed this block whatsoever. See my answer to Mike